infrastructure/main.tf

```diff
@@ -21,7 +21,7 @@ locals {
   s2s_url        = "http://rpe-service-auth-provider-${local.local_env}.service.core-compute-${local.local_env}.internal"
   dm_store_url   = "http://dm-store-${local.local_env}.service.core-compute-${local.local_env}.internal"
 
-  db_connection_options  = "?ssl=true"
+  db_connection_options  = "?ssl=true&sslmode=require"
```

Comment:
I think the jenkins library also needs a change for this too for the db migrate task to pass

Reply:
sharp. [pr](https://github.com/hmcts/cnp-jenkins-library/pull/283)